classes/locations.php

```diff
@@ -73,6 +79,7 @@ public function register_locations() {
 				'search_items'       => 'Search Locations',
 				'not_found'          => 'No Locations found',
 				'not_found_in_trash' => 'No Locations found in trash',
+				'menu_name'          => 'SimpleMap',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->